odbc/tests/common/mod.rs

```diff
@@ -0,0 +1,113 @@
+extern crate odbc_api;
+
+use self::odbc_api::handles::StatementImpl;
+use self::odbc_api::{Connection, Cursor, CursorImpl, Environment};
+use lazy_static::lazy_static;
+use odbc_api::ResultSetMetadata;
+use std::env;
+
+#[derive(Debug, PartialEq)]
+pub struct ResultsetMetadata {
```

Comment:
In order to keep the scope of this ticket more constrained, I'd suggest that we limit this ticket just to getting a basic handwritten C API test working and automated via evergreen, and split out the YAML-specified result-set tests into a separate ticket

Reply:
Done!
I tried to see how it would work with the yaml test definitions. I'll keep that on hand for later if needed.
I'm not too convince anyway still., because of how tests are generated in Rust.